tests/python/unittest/test_te_schedule.py

```diff
@@ -321,6 +321,33 @@ def test_legalize_invalid_attach():
     assert isinstance(stmt, tvm.tir.stmt.For)
 
 
+def test_compute_at():
+    def add():
+        shape = (16, 16)
+        A = tvm.te.compute(shape, lambda *i: 1.0, name="A")
+        B = tvm.te.compute(shape, lambda *i: 2.0, name="B")
+        C = tvm.te.compute(shape, lambda *i: A(*i) + B(*i), name="C")
+        return A, B, C
+
+    def invalid_compute_at_self():
+        A, B, C = add()
+        s = tvm.te.create_schedule(C.op)
+        s[C].compute_at(s[C], C.op.axis[0])
```

Comment:
Is there any way that we can just dismiss this operation?
It's fine to raise error here, but I'm thinking that if we can treat compute_at a axis to its self as no effect would be better.

Reply:
I would prefer throwing an error tho. If this statement is written by users, we should at least give a warning instead of dismissing it silently; otherwise users may not know this statement has to be fixed. If the statement is generated by auto-scheduler, we just need to catch the failure and throw the schedule away. Taking both cases into consideration, throwing an error is more reasonable, because we don't want to see lots of warnings during the tuning process.